src/Symfony/Component/BrowserKit/Cookie.php

```diff
@@ -92,12 +92,11 @@ public function __toString()
         if (null !== $this->expires) {
             $dateTime = \DateTime::createFromFormat('U', $this->expires, new \DateTimeZone('GMT'));
 
-            if ($dateTime === false) {
-                // this throw will provoke PHP fatal
-                throw new \UnexpectedValueException(sprintf('The cookie expiration time "%s" is not valid.'), $this->expires);
```

Comment:
since __toString cannot throw an exception, the validation of `$this->expires` should be done in the setter, i.e. the constructor in this case.